examples/scripts/import_export.ts

```diff
@@ -34,10 +34,10 @@ sayHello("World");
 import * as util from "./util.ts";
 util.sayHello("World");
 
-// Imports don't have to be relative, they can also reference absolute file,
-// https, or [JSR](https://jsr.io) URLs.
+// Imports don't have to be relative, they can also reference 
+// absolute file, npm, or [JSR](https://jsr.io) URLs.
```

Comment:
I’d argue keeping the https mentioned as it’s a feature in deno, just because deno.land/x is replaced with jsr doesn’t mean you still can’t use http imports for say esm.sh

Reply:
Fair. Agreed.